### PR TITLE
[MIG][12.0] delivery carrier partner

### PR DESCRIPTION
**Pull request by ValentinVinagre**
Standard Migration.
cc @sygel-technology

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Tested on runbot 👍

File: delivery_carrier_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Delivery carrier partner",
+    "summary": "Add a partner in the delivery carrier",
+    "version": "12.0.1.0.0",
+    "category": "Delivery",
```
Comment (rousseldenis) [inline delivery_carrier_partner/__manifest__.py:7]: Please set to 'Warehouse' as it would appear in wrong category on OCA shop

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

File: delivery_carrier_partner/__manifest__.py (hunk 1 above)
Comment (gurneyalex) [inline delivery_carrier_partner/__manifest__.py:7]:
```suggestion
    "category": "Warehouse",
```

**Pull request merged**